### DISCUSSION TITLE
Frequent "Unknown" API Response Status During Speech to Text (Streaming) Causing Malfunction, lang=ja

### DISCUSSION THREAD

**Discussion by roxx-akihiro-tanaka** [deepgram/community#783] [General help] [answered]
Dear Deepgram Team,

I am writing to inquire about a recurring issue we are experiencing with the Deepgram Speech to Text (Streaming) API, specifically for Japanese language transcription within our React application. We are frequently encountering "Unknown" API response statuses, which is causing our transcription process to malfunction.

This issue is significantly impacting our ability to accurately transcribe live Japanese audio streams within our React application. To better understand the root cause and implement a solution, we would appreciate it if you could provide us with further information.

Specifically, some recent occurrences were on:

- May 26, 2024, at 00:01:29
    - `request_id`: b78f52ee-30fc-4ce3-9b3f-7bfc7839e649
- May 25, 2024, at 23:56:58
    - `request_id`: 8ef14084-02f8-4cf0-909a-ed01671474d1
- May 25, 2024, at 23:44:08
    - `request_id`: ec575397-dc8c-4806-87bf-491e792feb7e
- May 25, 2024, at 23:32:15
    - `request_id`: 03164648-6c4b-4804-803f-2b6a7e144588

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@roxx-akihiro-tanaka  I checked into the logs of these requests and I noticed you didn't use our new Nova models, is there a reason for that?  [Nova-2 now supports the Japanese language](https://developers.deepgram.com/docs/models-languages-overview#nova-2) so try that model and see if you have a better experience. 

`nova-2` or `nova-2-general`
